Docs/MoltenVK_Runtime_UserGuide.md

```diff
@@ -219,6 +220,27 @@ To link **MoltenVK** to your application as a dynamic library (`.dylib`), follow
       - `MoltenVK/dylib/tvOS/libMoltenVK.dylib` *(tvOS)* 
      
 
+<a name="install_vksdk"></a>
+### Install *MoltenVK* replacing the Vulkan SDK .dylib
+
+There are a few potential issues when building **MoltenVK** to replace the version installed via the *[Vulkan SDK](https://vulkan.lunarg.com/sdk/home)* standard install process, which lives in `/usr/local/lib/libMoltenVK.dylib`.
+
+1. You must *remove* the existing `.dylib` file before copying the new one, because of the way that the gatekeeper system works to prevent malicious overwriting of files in standard locations such as `/usr/local`:
+
+```bash
+$ sudo rm /usr/local/lib/libMoltenVK.dylib
+$ sudo cp Package/Release/MoltenVK/dylib/macOS/libMoltenVK.dylib /usr/local/lib
+```
+
+If you do not do the remove first, your application will terminate immediately with a singularly unhelpful `Killed: 9` message.
+
+2. Do *not* copy the `MoltenVK_icd.json` file from the newly-built package to `/usr/local/share/vulkan/icd.d` -- it will not work and will result in errors about not being able to initialize the instance.  The one installed by Vulkan SDK uses a relative path to specify the location of the `.dylib`, whereas the one in the package specifies it in the same directory.
+
+3. The default config for command-line build has verbose logging info turned on -- if you want it to be like the original, use this command for building:
+
+```bash
+$ make macos MVK_CONFIG_LOG_LEVEL=1
+```
```

Comment:
Please text-wrap this content in a similar manner to the rest of the content in this document, so it's also readable in a regular text editor. Open the document in Xcode or a text editor to check the wrapping.

Reply:
Your instructions are great for devs just looking to build and use MoltenVK from the repo. Thanks for adding them.

But for future reference if you are developing and testing changes to MoltenVK, I rename the `libMoltenVK.dylib` file in `/usr/local/lib/` and then create a symlink there to `<path-to-MoltenVK>/Package/Latest/MoltenVK/dylib/macOS/libMoltenVK.dylib`. That way any rebuild of MoltenVK is automatically available. I just have to remember to do it each time I download and install a new SDK.